### DISCUSSION TITLE
React Element not working

### DISCUSSION THREAD

**Discussion by desrtCodr** [add2cal/add-to-calendar-button#459] [Q&A] [answered]
I'm working in Next / React. I can not get the button to display. Not sure what is the issue as the docs don't seem to mention anything additional that I need to do when using the React Wrapper. Perhaps its some silly CSS issue, hope someone can help me figure it out. Thanks so much. 

![image](https://github.com/add2cal/add-to-calendar-button/assets/48463280/13a63129-dd29-4369-96d6-acdc4ae7bfb0)
<img width="1215" alt="image" src="https://github.com/add2cal/add-to-calendar-button/assets/48463280/d2528f26-9cb1-4123-857a-b6ff1a59ef70">

**Comment by jekuer** [comment, marked as answer]
Hi @desrtCodr ,
Please check our Stackblitz example for Next.js at https://stackblitz.com/edit/add-to-calendar-button-next-js?file=components%2Fadd-to-calendar-button.js

Important: Next, per default, wants to prerender the button, which is not possible and will fail. You need to specify `ssr:false` or use any other method to tell Next that this is a client-only component.

Best,
Jens

**Reply by desrtCodr** [reply]
Making the component a client component worked as you stated. I appreciate your quick reply!
